Add a new doc explaining the Holdouts feature (#9925)
PostHog/posthog.com: contents/docs/experiments/holdouts.mdx

---
title: Holdouts
---

> Holdouts let you create persistent groups of users who can be excluded from multiple experiments or all experiments.

Reviewer: ```suggestion
When running A/B tests, you can create permanent lists of users called "holdouts" who won't participate in experiments. You can keep these same users out of either specific experiments or all of your testing entirely.
```

Reviewer: question actually: Can you specify which users in the holdout? Or is it random?

Reviewer 2: We now call it "Experiments", so let's use "When running experiments, ..."

Reviewer 2: @Lior539 Why use "permanent" instead of "persistent"? I think "persistent" fits better here.

To me, "permanent" suggests something that never changes. But holdouts can, and should, be lifted after a certain period. They persist across multiple experiments, but they aren't permanent because, at some point, the holdout will no longer be used.

Reviewer: you're right @jurajmajerik ! I think we can actually  remove the word "permanent/persistence" altogether. The sentence still makes sense then.

Also flagging in case it got missed:
> Can you specify which users are in the holdout? Or is it random?

Author: Applied a couple of edits 84dcf6042c017149185d3498321612d1bfbc0032, f6e682cc38a22d703b3614db0fe7013c17a036ee

> To create a new holdout, navigate to "Experiments" > "Holdout groups" and click "New holdout". Enter a descriptive name for your holdout, fill out the optional description to help you remember its purpose, and set the percentage of users you want to include in the holdout. We recommend between 1% and 10%; the holdout should be large enough to be statistically significant, but small enough as to not significantly reduce the pool of users available for experiments.

Reviewer: Let's also add a screenshot here. Use <ProductScreenshot> to upload both dark and light modes
```suggestion
To create a new holdout:

1. Click the [Holdout groups](add the link to the posthog app here e.g. [https://](https://us.posthog.com)/experiments/holdouts or whatever it is) in the experiments tab".
2. Click **New holdout**. 
3. Enter a descriptive name for your holdout, fill out the optional description to help you remember its purpose, and set the percentage of users you want to include in the holdout. 

The holdout should be large enough to be statistically significant, but small enough as to not significantly reduce the pool of users available for experiments. We recommend between 1% and 10%.
```

> Once created, you can assign the holdout when creating a new experiment, or while editing the experiment while it's in a draft state. The holdout becomes locked once any experiment using it is launched to prevent data inconsistencies that could affect results.

Reviewer: ```suggestion
Once created, you can assign the holdout to new or existing experiments. The holdout becomes locked once any experiment using it is launched and you will be unable to modify it. This is to prevent data inconsistencies that could affect results.
```

> When assigned to an experiment, your holdout will be treated as another variant in experiment analysis. You'll be able to see the conversion rate, delta %, credible interval, and win probability alongside other experiment variants.

Reviewer: Screenshot would be nice here too

---
title: Holdouts
---

When running experiments, you can create lists of users called "holdouts" who will be excluded. You can keep these same users out of either specific experiments or all of your testing entirely.

To create a new holdout:

> 1. Click the [Holdout groups](add the link to the posthog app here e.g. [https://](https://us.posthog.com)/experiments/holdouts or whatever it is) in the experiments tab".

Author: ```suggestion
1. Navigate to Experiments and click on the Holdout groups tab.
```
